### DISCUSSION TITLE
Error download meteofrance

### DISCUSSION THREAD

**Discussion by meb427** [open-meteo/open-meteo#614] [Q&A]
Hi,

When  trying to download meteo france

 openmeteo-api download-meteofrance arpege_world
[ INFO ] Downloading domain 'arpege_world' run '2024-01-19T12:00'
App/MeteoFranceDownloader.swift:94: Fatal error: Please specify environment variable 'METEOFRANCE_API_KEY'
Illegal instruction (core dumped)

And when checking code I see only one place where METEOFRANCE_API_KEY is referenced. How to get this API key and I guess I should place it in the env?

Edit: I guess it's here https://donneespubliques.meteofrance.fr/

Thanks in advance.

**Comment by patrick-zippenfenig** [comment]
Hi, yes you have to specify a MeteoFrance API key in the `METEOFRANCE_API_KEY` environment variable. An environment variable is used instead of a CLI option, because the API key can be a very long JWT token
